chemclipse/pom.xml

```diff
@@ -102,10 +102,10 @@
 					<encoding>${project.build.sourceEncoding}</encoding>
 					<extraClasspathElements>
 						<extraClasspathElement>
-							<groupId>com.oracle</groupId>
-							<artifactId>javafx</artifactId>
-							<version>8.0.0-SNAPSHOT</version>
-							<systemPath>${java.home}/lib/jfxswt.jar</systemPath>
+							<groupId>org.openjfx</groupId>
```

Comment:
Is this part of @adoptium?

Reply:
No, but http://blog.vogella.com/2019/11/15/add-javafx-controls-to-a-swt-eclipse-4-application-eclipse-rcp-cookbook-update/ suggested a p2 update site, so I went for that. This allows development on systems where @openjfx is not available such as @openSUSE.